python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -600,14 +614,14 @@ def __str__(self) -> str:
 @whitelist_for_serdes
 @record
 class AllSelection(AssetSelection):
-    include_sources: Optional[bool] = None
+    include_external_assets: Optional[bool] = None
```

Comment:
these classes are all whitelisted for serdes - is it safe to change the field name without back compat?

Reply:
@alangenfeld great catch: https://app.graphite.dev/github/pr/dagster-io/dagster/27757